src/index.ts

```diff
@@ -6465,7 +6465,7 @@ const shardusSetup = (): void => {
           ShardeumFlags.StakingEnabled && numActiveNodes >= ShardeumFlags.minActiveNodesForStaking
 
         //Checks for golden ticket
-        if (appJoinData.adminCert?.goldenTicket === true && appJoinData.mustUseAdminCert === true) {
+        if (appJoinData.adminCert?.goldenTicket === true || appJoinData.mustUseAdminCert === true) {
```

Comment:
We can eliminate the else section because adding the goldenTicket automatically sets the other parameter to true. However, if we retain the 2nd check and the goldenTicket is absent, the behavior remains unchanged, allowing admin cert nodes (without golden tickets) to join during processing mode.

Reply:
requested changes made and smoke testing done again.